### PR TITLE
Update Iliia name and workplace

### PR DESCRIPTION
**Pull request by hauleth**

### CODE REVIEW

**Comment by hauleth** [comment]
cc @deadtrickster

**Review by ferd — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -135,6 +135,6 @@ Approvers:
 - [Fred Hebert](https://github.com/ferd), Honeycomb
 
 Maintainers:
-- [Łukasz Jan Niemier](https://github.com/hauleth), Remote Technology, Inc.
-- [Ilya Khaprov](https://github.com/deadtrickster), KOBIL Systems
+- [Łukasz Jan Niemier](https://github.com/hauleth)
+- [Iliia Khaprov](https://github.com/deadtrickster), VMWare
```
Comment (deadtrickster) [inline README.md:139]: I didn't spell VMware right, haha. Could you please change VMWare -> VMware. Thank you!

**Review by deadtrickster — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tsloughter) [inline README.md:139]:
```suggestion
- [Iliia Khaprov](https://github.com/deadtrickster), VMware
```

**Comment by deadtrickster** [comment]
Great, thank you all :-)

**Pull request merged**